editor/src/core/model/__snapshots__/storyboard-utils.spec.ts.snap

```diff
@@ -0,0 +1,79 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`addStoryboardFileToProject adds storyboard file to project that does not have one 1`] = `
```

Comment:
could you make this an inline snapshot? I think we should consider moving away from sidecar snapshot files, because they make it too easy to capture too much information – meaning that the test will more likely fail for unrelated things. 
Also, inline snapshots are way more human readable, even if they are long, they are colocated with the test itself, no need to hunt them down by the name.

Reply:
That sounds like a wider decision though that we probably shouldn't make in a PR.